clang/test/Driver/android-ndk-standalone.cpp

```diff
@@ -14,7 +14,7 @@
 // CHECK: "-internal-externc-isystem" "{{.*}}/sysroot/usr/include/arm-linux-androideabi"
 // CHECK: "-internal-externc-isystem" "{{.*}}/sysroot/include"
 // CHECK: "-internal-externc-isystem" "{{.*}}/sysroot/usr/include"
-// CHECK: "{{.*}}ld{{(.exe)?}}" "--sysroot=[[SYSROOT:[^"]+]]"
+// CHECK: "{{.*}}ld.lld{{(.exe)?}}" "--sysroot=[[SYSROOT:[^"]+]]"
```

Comment:
These should just be `ld` to match what is in `clang/test/Driver/Inputs/basic_android_ndk_tree/arm-linux-androideabi/bin/`.  @DanAlbert and I discussed whether we can assume `lld` is the default since the Android toolchain defaults to lld but that's still undecided.